reana_server/utils.py

```diff
@@ -570,7 +574,7 @@ def get_workspace_retention_rules(
                     "Maximum workflow retention period was reached. "
                     f"Please use less than {WORKSPACE_RETENTION_PERIOD} days."
                 )
-            # TODO: add validation for rules
+            validate_retention_rule(rule)
```

Comment:
(minor) what do you think about moving these lines inside `validate_retention_rule`?
https://github.com/reanahub/reana-server/blob/1ac23c4729bc2b2443ddf1b1f6dbb6d14edbc1a8/reana_server/utils.py#L568-L572